es-ds-docs/nuxt.config.ts

```diff
@@ -0,0 +1,30 @@
+// https://nuxt.com/docs/api/configuration/nuxt-config
+export default defineNuxtConfig({
+    compatibilityDate: '2024-04-03',
+
+    // https://nuxt.com/docs/getting-started/styling#the-css-property
+    css: [
+        '@/assets/scss/main.scss',
```

Comment:
Per previous comment, this could become `'@energysage/es-bs-base/scss/bootstrap.scss'`?

Reply:
Done!